core/src/backends/eclair/rest/node.rs

```diff
@@ -14,22 +14,19 @@ pub struct EclairRest {
 }
 
 impl EclairRest {
-    pub fn new(config: NodeConfig) -> Result<Self, Error> {
-        let config: EclairRestConfig = config.into();
-
-        let encoded_auth = format!("{}:{}", config.username, config.password);
-        let auhtorization = format!("Basic {}", base64::encode(encoded_auth));
+    pub fn new(config: EclairRestConfig) -> Result<Self, Error> {
+        let auth = format!("{}:{}", &config.username, &config.password);
+        let authorization = format!("Basic {}", base64::encode(encoded_auth));
 
         let mut headers = reqwest::header::HeaderMap::new();
-        let auhtorization_value = reqwest::header::HeaderValue::from_str(&auhtorization).unwrap();
+        let auhtorization_value = reqwest::header::HeaderValue::from_str(&auhtorization)?;
```

Comment:
```suggestion
        let authorization_value = reqwest::header::HeaderValue::from_str(&authorization)?;
```
Variable name changed in e55e0e4fb20d951fd6079d2166eb18579eca5344.